spec/controllers/chapters_controller_spec.rb

```diff
@@ -366,6 +358,28 @@
         it_redirects_to_with_error(work_path(work), "Sorry, you don't have permission to access the page you were trying to reach.")
       end
     end
+
+    context "when co-creator is logged in" do
+      before do
+        @co_creator = create(:user)
```

Comment:
We should use let to avoid sharing state between tests in instance variables. You don't need to update the existing instance variables though.

Reply:
will do :+1:  mini-commits on branch shouldn't be too noisy since the commits get squashed before merge?

Reply:
Yeah more commits is fine, we always use squash and merge.